app/views/layouts/_login.html.erb

```diff
@@ -9,7 +9,7 @@
 
 <hr class="hidden-lg hidden-md"/>
 
-<div class="col-md-6">
+<div class="login-form" class="col-md-6">
```

Comment:
Is this as reliable as `class="login-form col-md-6"`? No idea whether it's treated differently, but seems like the kind of thing to break in old browsers, for example. Can't find any info on what happens when an attribute is defined multiple times on an element like this.